gef.py

```diff
@@ -2150,7 +2150,7 @@ def checksec(filename: str) -> Dict[str, bool]:
     return Elf(filename).checksec
 
 
-@lru_cache()
+@deprecated("Replaced by `reset_architecture` instead")
```

Comment:
In this case, it should be replaced by `gef.arch`
```suggestion
@deprecated("Use `gef.arch` instead")
```